styles/globals.css

```diff
@@ -1264,6 +1264,51 @@ div.image .img {
   margin-bottom: 10em;
 }
 
+.compact {
```

Comment:
Could you please add a `media print` css query to exclude the webiste navbar, footer and other useless elements while printing the schedule?